net-mgmt/zabbix5-proxy/Makefile

```diff
@@ -0,0 +1,8 @@
+PLUGIN_NAME=		zabbix5-proxy
+PLUGIN_VERSION=		1.3
+PLUGIN_COMMENT=		Zabbix Proxy enables decentralized monitoring
+PLUGIN_DEPENDS=		zabbix5-proxy
+PLUGIN_CONFLICTS=   zabbix4-proxy
```

Comment:
would you mind adding the conflict on the other end as well?

Reply:
Hope it's ok when I do this in one PR? :)

Reply:
sure, thanks!